src/components/StatsScoringDialog.vue

```diff
@@ -17,73 +17,66 @@
-        Spades. At the end of each season, the top 8 players play a double elimination championship tournament
-        and the
-        <strong>champions are permanantly accoladed on the site.</strong>
+        {{ t('game.dialogs.statsScoringDialog.competitiveCuttle.description') }}
+        <strong>{{ t('game.dialogs.statsScoringDialog.competitiveCuttle.highlight') }}</strong>
       </p>
       <div class="d-flex justify-space-around flex-wrap mt-4">
         <AwardCard username="Champion player" :place="1" class="mb-4" />
         <AwardCard username="Second Place Player" :place="2" class="mb-4" />
         <AwardCard username="Third Place Player" :place="3" class="mb-4" />
       </div>
       <p>
-        Each Season is divided into 13 weeks. For each week, we count the number of best 2/3 ranked matches
-        each player wins against unique opponents (ignoring stalemates) and assign players points based on
-        their weekly standing.
+        {{ t('game.dialogs.statsScoringDialog.seasonDescription') }}
       </p>
       <v-list class="mt-4" bg-color="surface-2" base-color="surface-1">
         <v-list-item>
           <v-chip variant="elevated" class="mr-2 mb-1" :color="theme.firstPlace">
-            5 Points
+            5 {{ t('game.dialogs.statsScoringDialog.points.displayName') }}
           </v-chip>
-          The player with the most wins gets 5 points for the week
+          {{ t('game.dialogs.statsScoringDialog.points.description.5Points') }}
         </v-list-item>
         <v-list-item>
           <v-chip variant="elevated" class="mr-2 mb-1" :color="theme.secondPlace">
-            4 Points
+            4 {{ t('game.dialogs.statsScoringDialog.points.displayName') }}
           </v-chip>
-          The player with the 2nd most wins gets 4 points for the week
+          {{ t('game.dialogs.statsScoringDialog.points.description.4Points') }}
         </v-list-item>
         <v-list-item>
           <v-chip variant="elevated" class="mr-2 mb-1" :color="theme.thirdPlace">
-            3 Points
+            3 {{ t('game.dialogs.statsScoringDialog.points.displayName') }}
           </v-chip>
-          The player with the 3rd most wins gets 3 points for the week
+          {{ t('game.dialogs.statsScoringDialog.points.description.3Points') }}
         </v-list-item>
         <v-list-item>
           <v-chip variant="outlined" class="mr-2 mb-1" :color="theme.primary">
-            2 Points
+            2 {{ t('game.dialogs.statsScoringDialog.points.displayName') }}
           </v-chip>
-          Each other player who won at least one match gets 2 points for the week
+          {{ t('game.dialogs.statsScoringDialog.points.description.2Points') }}
         </v-list-item>
         <v-list-item>
           <v-chip variant="outlined" class="mr-2 mb-1" color="#000">
-            1 Point
+            1 {{ t('game.dialogs.statsScoringDialog.points.singular') }}
```

Comment:
We actually don't need a separate block for this, we can use i18n's pluralization system to specify the count like so 
```suggestion
            {{ t('game.dialogs.statsScoringDialog.points.displayName', 1) }}
```

Then we can correspondingly update all the other references to use this new definition and update the definition itself in the language files to specify the count and to use a different word for the singular version

Reply:
You can see an example of this in ScoreGoalTooltip.vue in the `kingPoints` computed property which pluralizes the `game.score.kings` string